src/builder.rs

```diff
@@ -167,6 +176,7 @@ impl AstarteOptions {
         Ok(self)
     }
 
+    /// Clone the astarte options.
```

Comment:
Maybe is better something like `Returns a reference to the AstarteOptions being constructed.`

Reply:
True. 
By the way, what is the purpose of this method?
The name `build` seems to imply some complex operation, while it just clones the options.
Why not let the user take care of cloning?